static/js/components/VerifyButton.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Button from 'react-bootstrap/lib/Button';
+
+class VerifyButton extends React.Component {
+  render() {
+    return <Button bsStyle="success">UPGRADE TO VERIFIED</Button>;
+  }
+}
+
+VerifyButton.PropTypes = {
+};
+
+export default VerifyButton;
```

Comment:
for this component (and the above component) are we anticipating needing to add more functionality to the buttons themselves?